runzi/execute/openquake/oq_hazard_task.py

```diff
@@ -69,10 +69,19 @@ def _setup_automation_task(self, task_arguments, job_arguments, environment, tas
         print("task arguments ...")
         print(task_arguments)
         print("=" * 50)
+        srm_logic_tree = task_arguments["hazard_model-srm_logic_tree"]
```

Comment:
these implicitly flattened dicts are confusing (completely my doing)! I wonder if there's a better way either, a) more logical and explicit dict keys or b) a different data type

Reply:
It does make it easier to store it and display it. But of course the structure is different from what's in runzi. I don't think it's confusing at all.